modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/notification/sender/v2/model/Authentication.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.rest.api.server.notification.sender.v2.model;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Authentication configuration for SMS sender
+ */
+public class Authentication {
+    
+    /**
+     * Authentication type enum
+     */
+    @JsonAdapter(TypeEnum.Adapter.class)
+    public enum TypeEnum {
+        @SerializedName("CLIENT_CREDENTIAL")
+        CLIENT_CREDENTIAL("CLIENT_CREDENTIAL"),
+        @SerializedName("BASIC")
+        BASIC("BASIC"),
+        @SerializedName("API_KEY")
+        API_KEY("API_KEY"),
+        @SerializedName("BEARER")
+        BEARER("BEARER"),
+        @SerializedName("NONE")
+        NONE("NONE");
+
+        private String value;
+
+        TypeEnum(String value) {
+            this.value = value;
```

Comment:
formatting issue, there should be newline in between method declaration and implementation. Check the other places as well

Reply:
https://github.com/wso2/product-is/pull/26329#discussion_r2591591133